getting-started/developers-guide.md

```diff
@@ -25,6 +25,7 @@ layout: getting-started
   - [from JavaScript code](https://developer.mozilla.org/en-US/docs/WebAssembly/Loading_and_running)
   - [as a CLI application](https://github.com/bytecodealliance/wasmtime/blob/master/docs/WASI-tutorial.md)
   - [from Node.js (with access to system resources)](https://nodejs.org/api/wasi.html)
+  - [from Deno](https://deno.land/manual/getting_started/webassembly)
```

Comment:
Link is invalid.

Reply:
Whoops sorry about that! Thanks for checking. Should be fixed now.